config/example_artillery_genius_pro_2023.cfg

```diff
@@ -0,0 +1,153 @@
+[include mainsail.cfg]
+[mcu]
+serial: /dev/serial/by-id/[your_id]
+# [printer]
+# kinematics: none
+# max_velocity: 1000
+# max_accel: 1000
+[stepper_x]
+step_pin: !PB14
+dir_pin: PB13
+enable_pin: !PB15
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA2
+position_endstop: 0
+position_max: 220
+homing_speed: 50
+second_homing_speed: 10
+[stepper_y]
+step_pin: PB10
+dir_pin: PB2
+enable_pin: !PB12
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA1
+position_endstop: 0
+position_max: 220
+homing_speed: 50
+second_homing_speed: 10
+[stepper_z]
+step_pin: PB0
+dir_pin: !PC5
+enable_pin: !PB1
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_max: 250
+homing_speed: 10
+second_homing_speed: 2
+[extruder]
+step_pin: PA7
+dir_pin: PA6
+enable_pin: !PC4
+microsteps: 16
+rotation_distance: 20.925
+gear_ratio: 66:22
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC9
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC0
+control: pid
+pid_kp: 20.860
+pid_ki: 1.131
+pid_kd: 96.218
+min_temp: 0
+max_temp: 250
+[heater_bed]
+heater_pin: PA8
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC1
+control: pid
+pid_kp: 42.365
+pid_ki: 0.545
+pid_kd: 822.940
+min_temp: 0
+max_temp: 130
+[fan]
+pin: PC8
+off_below: 0.1
+[heater_fan extruder]
+pin: PC7
+off_below: 0.2
+[controller_fan case]
+pin: PC6
+off_below: 0.3
+idle_speed: 0.0
+[temperature_sensor mainboard]
```

Comment:
Please do not add custom temperature sensors for mcu's and hosts, these may not be available on the end users equipment

Reply:
Hi, they does work perfectly on Artillery genius .